internal/datastore/postgres/gc.go

```diff
@@ -67,6 +67,9 @@ func (pgd *pgDatastore) TxIDBefore(ctx context.Context, before time.Time) (datas
 		return datastore.NoRevision, err
 	}
 
+	// Force the proper index for the GC operation.
+	sql = "/*+ IndexOnlyScan(" + schema.TableTransaction + " " + schema.IndexSortedRelationTupleTransaction.Name + ") */" + sql
```

Comment:
Please clarify in the PR body that `pg_hint_plan` receives index hints as SQL comments

Reply:
please turn this into a helper function